build_stack.sh

```diff
@@ -188,12 +188,12 @@ build_lib sigio
 build_lib sfcio
 build_lib gfsio
 build_lib w3nco
+build_lib nemsio
```

Comment:
This needs to come after w3emc.

Reply:
Some of the changes were to build  the hpc-stack with w3emc/2.7.3 as needed for the SRW release/public-v2. The "develop" branch has an update w3emc/2.9.2. 
Will revise the changes and break them into smaller PRs.